app/services/katello/pulp3/repository/apt.rb

```diff
@@ -31,22 +31,19 @@ def sync_url_params(sync_options)
         end
 
         def mirror_remote_options
-          super.merge(
-            {
-              distributions: repo.deb_releases + "#{' default' unless repo.deb_releases.include? 'default'}"
-            }
-          )
+          distributions = "#{repo.deb_releases}"
+          if Setting['deb_use_simple_publish'] and not distributions.split(' ').include? 'default'
+            distributions.concat(' default')
+          end
+
+          super.merge({policy: policy, distributions: distributions})
```

Comment:
```suggestion
          super.merge({distributions: distributions})
```
This is a rebase error.